src/components/PaginaComprador.js

```diff
@@ -0,0 +1,237 @@
+import React from "react";
+import axios from "axios";
+import styled from "styled-components";
+
+const MainContainer = styled.div`
+  display: flex;
+`;
+const Filtros = styled.div`
+  background: #495af6;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  width: 250px;
+  margin: 15px 95px;
+  font-family: "Poppins", sans-serif;
+  border-radius: 20px;
+
+  label {
+    font-size: 20px;
+  }
+  select {
+    height: 40px;
+    border-radius: 15px;
+    outline: none;
+    border: none;
+    margin: 10px;
+    font-family: "Poppins", sans-serif;
+  }
+
+  input {
+    height: 40px;
+    outline: none;
+    border: none;
+    margin: 10px;
+    border-radius: 15px;
+    font-family: "Poppins", sans-serif;
+  }
+`;
+
+export class PaginaComprador extends React.Component {
+  state = {
+    carros: [],
+    inputMinimo: "",
+    inputMaximo: "",
+    inputNome: "",
+    ordernarPor: "nome",
+    ordem: "crescente",
+  };
+
+  componentDidMount() {
+    this.pegarCarros();
+  }
+
+  pegarCarros = async () => {
+    try {
+      const response = await axios.get(
+        "https://us-central1-labenu-apis.cloudfunctions.net/futureCarThree/cars"
+      );
+      this.setState({ carros: response.data.cars });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  inputMinimo = (event) => {
+    this.setState({ inputMinimo: event.target.value });
+  };
+
+  inputMaximo = (event) => {
+    this.setState({ inputMaximo: event.target.value });
+  };
+
+  inputNome = (event) => {
+    this.setState({ inputNome: event.target.value });
+  };
+
+  ordenarPor = (event) => {
+    this.setState({ ordernarPor: event.target.value.toLowerCase() });
+  };
+
+  ordem = (event) => {
+    this.setState({ ordem: event.target.value.toLowerCase() });
+  };
+
+  render() {
+    const listaFiltrada = this.state.carros.filter((carro) => {
+      if (
+        this.state.inputMinimo !== "" &&
```

Comment:
Os filtros de preço estão com um comportamento meio estranho, nem sempre funcionam. Acredito que para funcionar seria só converter o `this.state.inputMinimo` para number antes de fazer operações 😄 

Além disso, fazer com que o input só aceite números (sendo do type=number) também seria interessante